user_docs/en/changes.t2t

```diff
@@ -15,6 +15,7 @@ What's New in NVDA
 == Changes ==
 - NVDA no longer supports Windows 7 and Windows 8.
 Windows 8.1 is the minimum Windows version supported. (#15544)
+- The audio output device and ducking mode options have been removed from the "Select Synthesizer" dialog. They can be found in the audio settings panel which can be opened with ``NVDA+control+u``. (#15512)

```

Comment:
can this be moved to the bottom of the list? component updates should be second.

Also please split into new lines

```suggestion
- The audio output device and ducking mode options have been removed from the "Select Synthesizer" dialog.
They can be found in the audio settings panel which can be opened with ``NVDA+control+u``. (#15512)
```